docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-reset.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: switch
```

Comment:
@Anita-ihuman I think the title here should be `reset` 👀
As it will be the one which will be shown here at, 

![image](https://user-images.githubusercontent.com/23498248/119568758-78f0f480-bdcb-11eb-89dd-181fb93eabb2.png)

Could you change it to reset?